### ISSUE TITLE
Reduce number of wallets displayed using `open wallet` option

### ISSUE THREAD

**Issue by prayank23** [bitcoin-core/gui#304]
**Is your feature request related to a problem? Please describe.**

I created lot of wallets to test something else related to https://github.com/bitcoin/bitcoin/issues/21510

![image](https://user-images.githubusercontent.com/13405205/116490651-8e026280-a8b5-11eb-9e0b-55af493fa2ca.png)

Wanted to check how GUI works after having so many wallets and first thing I noticed is the time it takes to display wallets for "Open Wallet" option, number of wallets and the whole screen covered with these names. It doesn't even include all the names and maybe they have gone out of the screen on right side.

![image](https://user-images.githubusercontent.com/13405205/116490599-66ab9580-a8b5-11eb-81f7-c9b95e70a1ca.png)

**Describe the solution you'd like**
Only display name for default wallet and maybe 9 other wallets based on some criteria. Provide an option to mention name/path for other wallets to open.

**Describe alternatives you've considered**
Move the wallets to a different location.

**Comment by jarolrod** [comment]
The GUI can't be designed with every single use case in mind, this can be considered an edge case. 

If a user created this many wallets, we need to show them. It would be extremely surprising and annoying for most of someone's wallets not show up.

Besides that, there are performance issues and that can be looked into.

**Comment by prayank23** [comment]
> If a user created this many wallets, we need to show them. 

All the wallets are not shown. Only what fits on one screen. If we need to show all wallets I think there are better ways to show it.

> It would be extremely surprising and annoying for most of someone's wallets not show up.

Normally lot of softwares show last few used files or projects in such lists and not everything. But provide an option to open other files. Or atleast arrange the list of files in a way that doesn't look ugly or uses whole screen to show some of them.

**Comment by jarolrod** [comment]
I don't think we should compare wallet files against, say, text files on a word processor. I think all wallets you create/import should be accessible. You shouldn't have to go through steps to find it and open it.

Things that can be improved, if there is consensus, is how we show the wallets list when there are many of them and the user has clicked the `Open Wallet` menu options.

How does this look when you click on the dropdown menu to load wallets?

**Comment by prayank23** [comment]
> How does this look when you click on the dropdown menu to load wallets?

Which dropdown?
